lib/Controller/PublicationsController.php

```diff
@@ -19,6 +20,7 @@
 use OCP\AppFramework\Http\TemplateResponse;
 use OCP\AppFramework\Http\JSONResponse;
 use OCP\AppFramework\OCS\OCSBadRequestException;
+use Doctrine\DBAL\Exception\NotNullConstraintViolationException;
```

Comment:
Dit is een use die je niet gebruikt in je nieuwe code, was je deze eerder vergeten toe te voegen? zelfde voor MetaDataService